### PR TITLE
feat: Introduce 'NEW INVITE' tag functionality for newly invited teams.

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
Introduce a feature called the 'NEW INVITE' tag functionality, designed specifically for teams that have recently been invited. This feature aims to enhance the user experience by visually highlighting and categorizing teams that have received recent invitations, thereby providing a clear and distinguishable marker for newly invited teams within the system.
Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

**Review by LordNayan — COMMENTED** [review]
Why are we using a different api and tag for identifying a new invite. Can we try to use created date to activate/deactivate this tag on the frontend itself.

@itsmdasifraza

**Comment by itsmdasifraza** [comment]
> Why are we using a different api and tag for identifying a new invite. Can we try to use created date to activate/deactivate this tag on the frontend itself.
> 
> @itsmdasifraza

We don`t have to show "NEW INVITE" with respect to no of days, We have to disable it once user enters that team.

File: src/modules/common/models/team.model.ts [hunk 1]
```diff
@@ -77,6 +78,10 @@ export class Team {
   @IsString()
   @IsOptional()
   updatedBy?: string;
+
+  @IsBoolean()
+  @IsOptional()
+  isNewInvite?: boolean;
```
Comment (LordNayan) [inline src/modules/common/models/team.model.ts:84]: Remove it from Team Model. Team Dto alone should be fine.

**Review by LordNayan — COMMENTED** [review]
Small comment added

File: src/modules/common/models/team.model.ts (hunk 1 above)
Comment (itsmdasifraza) [reply src/modules/common/models/team.model.ts:86]: resolved

**Review by LordNayan — APPROVED** [review]

**Pull request merged**